cli/command/load_balancer.go

```diff
@@ -10,6 +18,497 @@ func NewLoadBalancerCommand(b *CommandBase) *LoadBalancerCommand {
+		LoadBalancerName: args["LOADBALANCER_NAME"],
+		EnvironmentID:    environmentID,
+		IsPublic:         !c.Bool("private"),
+		Ports:            ports,
+		HealthCheck:      healthCheck,
+	}
+
+	jobID, err := l.client.CreateLoadBalancer(req)
+	if err != nil {
+		return err
+	}
+
+	onCompleteFn := func(loadBalancerID string) error {
+		loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+		if err != nil {
+			return err
+		}
+
+		l.printer.StopSpinner()
+		return l.printer.PrintLoadBalancers(loadBalancer)
+	}
+
+	return l.waitOnJobHelper(c, jobID, "Creating Load Balancer", onCompleteFn)
+}
+
+func (l *LoadBalancerCommand) delete(c *cli.Context) error {
+	return l.deleteHelper(c, "loadbalancer", func(loadBalancerID string) (string, error) {
+		return l.client.DeleteLoadBalancer(loadBalancerID)
+	})
+}
+
+func (l *LoadBalancerCommand) dropport(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "LOADBALANCER_NAME", "HOST_PORT")
+	if err != nil {
+		return err
+	}
+
+	port, err := strconv.ParseInt(args["HOST_PORT"], 10, 64)
+	if err != nil {
+		return fmt.Errorf("'%s' is not a valid integer", args["HOST_PORT"])
+	}
+
+	loadBalancerID, err := l.resolveSingleEntityIDHelper("load_balancer", args["LOADBALANCER_NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+	if err != nil {
+		return err
+	}
+
+	var exists bool
+	for i, p := range loadBalancer.Ports {
+		if p.HostPort == port {
+			loadBalancer.Ports = append(loadBalancer.Ports[:i], loadBalancer.Ports[i+1:]...)
+			exists = true
+		}
+	}
+
+	if !exists {
+		return fmt.Errorf("Host port '%v' doesn't exist on this Load Balancer", port)
+	}
+
+	req := models.UpdateLoadBalancerRequest{
+		LoadBalancerID: loadBalancerID,
+		Ports:          &loadBalancer.Ports,
+		HealthCheck:    nil,
+	}
+
+	jobID, err := l.client.UpdateLoadBalancer(req)
+	if err != nil {
+		return err
+	}
+
+	onCompleteFn := func(loadBalancerID string) error {
+		loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+		if err != nil {
+			return err
+		}
+
+		l.printer.StopSpinner()
+		return l.printer.PrintLoadBalancers(loadBalancer)
+	}
+
+	return l.waitOnJobHelper(c, jobID, "Dropping port from load balancer", onCompleteFn)
+}
+
+func (l *LoadBalancerCommand) healthcheck(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "LOADBALANCER_NAME")
+	if err != nil {
+		return err
+	}
+
+	requiresUpdate := false
+	healthCheck := models.HealthCheck{}
+
+	if target := c.String("healthcheck-target"); target != "" {
+		target, err := parseTarget(target)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Target = target
+	}
+
+	if interval := c.String("healthcheck-interval"); interval != "" {
+		interval, err := strconv.Atoi(interval)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Interval = interval
+	}
+
+	if timeout := c.String("healthcheck-timeout"); timeout != "" {
```

Comment:
How do we check for an invalid input then given:
```
func TestUpdateHealthCheck_UserInputError(t *testing.T) {
     base, ctrl := newTestCommand(t)
     defer ctrl.Finish()
 
     contexts := map[string]*cli.Context{
         "Non-int '--healthcheck-interval' flag": getCLIContext(t, args{"lname"}, flags{"healthcheck-interval": "two"}),
         "Non-int '--healthcheck-timeout' flag":  getCLIContext(t, args{"lname"}, flags{"healthcheck-timeout": "two"}),
         "Non-int '--healthy-threshold' flag":    getCLIContext(t, args{"lname"}, flags{"healthy-threshold": "two"}),
         "Non-int '--unhealthy-threshold' flag":  getCLIContext(t, args{"lname"}, flags{"unhealthy-threshold": "two"}),
         "Missing NAME arg":                      getCLIContext(t, nil, flags{"interval": 2}),
     }
 
     loadBalancerCommand := NewLoadBalancerCommand(base.Command())
     for name, c := range contexts {
         if err := loadBalancerCommand.healthcheck(c); err == nil {
             t.Fatalf("%s: error was nil!", name)
         }
     }
 }
```

Reply:
I ask this because c.Int() would return only an int (in this case, 0) if the value isn't found (as opposed to being able to return a value and an `error`.